### DISCUSSION TITLE
Inquiries made by email

### DISCUSSION THREAD

**Discussion by kiri03** [opengisch/QField#3965] [Q&A]
Yesterday I made an inquiry by email about the invoices issued to the addresses:
hello@qfield.cloud
beta-transition@qfield.cloud

I hope to have an answer soon or, if it is not possible, could you indicate another email address where I can contact you?

Thanks

**Comment by lucienicolier** [comment]
Hi, 

did you receive an answer?

Now you can ask a question directly on qfield.cloud by clicking on the question mark down-right.

Best

**Reply by cez2020** [reply]
That's a chat bot, though

**Reply by lucienicolier** [reply]
If he can't answer, it is going to the ticketing system.

**Comment by cez2020** [comment]
My organization is about to use QField for the implementation of a data collection project. We have a series of specific questions about the app, have sent out an email, and have been basically ghosted. Our main issue now is that we were counting on the 100GB QField came with, and now storage has been reduced to only 100MB. We are interested in upgrading but would like to speak to someone in QField about our data requirements. @lucienicolier

**Reply by lucienicolier** [reply]
Hi @cez2020,
did you receive your answers from OPENGIS.ch team? 
If you need additional information, please let me know.
